### DISCUSSION TITLE
Can't get the AggregatorV3Interface.sol in Chainlink github repo

### DISCUSSION THREAD

**Discussion by NomanALPHA007** [Cyfrin/foundry-full-course-cu#1563] [General]
<img width="1241" alt="Screenshot 2024-04-02 050156" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/112021296/0eea9c7a-1d2d-4d94-85b8-53e4be307c99">
Can't get the AggregatorV3Interface.sol in Chainlink github repo.
Currently I'm doing Solidity Fundamentals section 2(FundMe) ,Lesson no 7. If someone can guide me i will be greatful

**Comment by EngrPips** [comment]
Hello @NomanALPHA007 , did you install the exact version of the chainlink repo installed in the lesson ?
